### ISSUE TITLE
Auto-upper case doesn't work 

### ISSUE THREAD

**Issue by radium0022** [TablePlus/TablePlus#800]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
Any. 
2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
150 
3. The steps to reproduce this issue:

A. Select auto-upper case key words option in the Query menu (located on the bottom left corner of the query tab).

B. The key words are not auto-upper cased automatically. 

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by huyphams** [comment]
Hi @radium0022  auto-upper doesn't work that way, it's only applied when typing.
if you want to upper all the current keyword, you can use the Beautify button.

**Comment by radium0022** [comment]
Hey, 

Thanks for the swift answer. Just to fully understand, do you mean that auto-upper is only working when using the auto complete function with tab? 

Because when I am typing all the sql statements are remaining in lower case, at the exeption of when I use tab to auto-complete them. 

Jean

**Comment by huyphams** [comment]
Yup, that’s true, it only works with auto completion popup. Sorry for the confusion.
